x-pack/plugins/drilldowns/public/service/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './drilldown_service';
```

Comment:
nit: would prefer explicit exports

Reply:
Let's leave it like that for now and discuss over Zoom how we want to proceed.